src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -190,6 +193,13 @@ public boolean publish(SlackRequest slackRequest) {
                 json.put("unfurl_links", "true");
                 json.put("unfurl_media", "true");
 
+                String apiEndpoint = "chat.postMessage";
+
+                if (StringUtils.isNotEmpty(updateMessage)) {
+                    json.put("ts", updateMessage);
```

Comment:
Call what it is, timestamp ⌚
```suggestion
                    json.put("ts", timestamp);
```

Reply:
Yup good point, i'll change it